src/amo/components/AddonMoreInfo/index.js

```diff
@@ -0,0 +1,169 @@
+          {i18n.gettext('Support Site')}
+        </a></li>
+      );
+    }
+
+    return (
+      <dl className="AddonMoreInfo-contents">
+        {homepage || supportUrl ? (
+          <dt className="AddonMoreInfo-links-title">
+            {i18n.gettext('Add-on Links')}
+          </dt>
+        ) : null}
+        {homepage || supportUrl ? (
+          <dd className="AddonMoreInfo-links-contents">
+            <ul className="AddonMoreInfo-links-contents-list">
+              {homepage}
+              {supportUrl}
+            </ul>
+          </dd>
+        ) : null}
+        <dt>{i18n.gettext('Version')}</dt>
+        <dd className="AddonMoreInfo-version">
+          {addon.current_version.version}
+        </dd>
+        <dt>{i18n.gettext('Last updated')}</dt>
+        <dd>
+          {i18n.sprintf(
+            // L10n: This will output, in English:
+            // "2 months ago (Dec 12 2016)"
+            i18n.gettext('%(timeFromNow)s (%(date)s)'), {
+              timeFromNow: i18n.moment(addon.last_updated).fromNow(),
+              date: i18n.moment(addon.last_updated).format('ll'),
+            }
+          )}
+        </dd>
+        {addon.current_version.license ? (
+          <dt className="AddonMoreInfo-license-title">
+            {i18n.gettext('License')}
+          </dt>
+        ) : null}
+        {addon.current_version.license ? (
+          <dd>
+            <a
+              className="AddonMoreInfo-license-link"
+              href={addon.current_version.license.url}
+            >
+              {addon.current_version.license.name}
+            </a>
+          </dd>
+        ) : null}
+        {addon.has_privacy_policy ? (
+          <dt className="AddonMoreInfo-privacy-policy-title">
+            {i18n.gettext('Privacy Policy')}
+          </dt>
+        ) : null}
+        {addon.has_privacy_policy ? (
+          <dd>
+            <Link
+              className="AddonMoreInfo-privacy-policy-link"
+              href={`/addon/${addon.slug}/privacy/`}
+            >
+              {i18n.gettext('Read the privacy policy for this add-on')}
+            </Link>
+          </dd>
+        ) : null}
+        {addon.has_eula ? (
+          <dt className="AddonMoreInfo-eula-title">
+            {i18n.gettext('End-User License Agreement')}
+          </dt>
+        ) : null}
+        {addon.has_eula ? (
+          <dd>
+            <Link
+              className="AddonMoreInfo-eula-link"
+              href={`/addon/${addon.slug}/eula/`}
+            >
+              {i18n.gettext('Read the license agreement for this add-on')}
+            </Link>
+          </dd>
+        ) : null}
+        {addon.id ? (
+          <dt
+            className="AddonMoreInfo-database-id-title"
+            title={i18n.gettext(`This ID is useful for debugging and
+              identifying your add-on to site administrators.`)}
+          >
+            {i18n.gettext('Internal Database ID')}
```

Comment:
Not sure if there was other guidance on naming but maybe we could avoid the use of Internal and Database they make it look a little like something that shouldn't be public. Would `Add-on ID` or just simply `ID` or `Identifier`.

Reply:
"Site Identifier" maybe?

Reply:
> "Site Identifier" maybe?

Works for me